.gitlab-ci.yml

```diff
@@ -40,7 +40,7 @@ variables:                         &default-vars
   ARCH:                            "x86_64"
   CI_IMAGE:                        "paritytech/ci-linux:production"
   # FIXME set to release
-  CARGO_UNLEASH_INSTALL_PARAMS:    "--version 1.0.0-alpha.11"
+  CARGO_UNLEASH_INSTALL_PARAMS:    "--git https://github.com/paritytech/cargo-unleash"
```

Comment:
it's ironic the tool aimed to fix versioning is unversioned

Reply:
We agreed with @gnunicorn to release a new version once we ensure that it's capable of fixing this versioning issue here.

Should we do that and switch to the new version, then?

Reply:
There was a release of `.alpha.12` in the meantime that's upgraded to the new Cargo so I switched to use that instead for now